### DISCUSSION TITLE
Unexpected token error in storefront

### DISCUSSION THREAD

**Discussion by MattWithCurls** [saleor/saleor#10187] [Q&A]
When running this Saleor CLI storefront create command once everything is done this is the error I'm getting Unexpected token < in JSON at position 0. Any help will be appreciated to fix this error.
![saleor](https://user-images.githubusercontent.com/36450476/179616331-1fcbbf35-5513-4c60-a19e-707372f546a5.png)
.

Thanks in advance

**Comment by karolkielecki** [comment]
Hi @MattWithCurls, which version of the CLI tool are you using? Please let me know, and I'll check it out.

**Reply by MattWithCurls** [reply]
Hi @karolkielecki using v1.9.0.

**Reply by karolkielecki** [reply]
So, did you use the following `saleor storefront create --demo` command? I've just created a storefront with that command without any errors. See the screenshot below. Could you please check the dev console to see if there are more errors? 

![image](https://user-images.githubusercontent.com/57907951/179970010-7531757e-645e-48db-8db5-3f10003dac79.png)

**Reply by MattWithCurls** [reply]
It's working perfectly now updated the CLI and the sandbox.
Thanks @karolkielecki

**Reply by karolkielecki** [reply]
Great, enjoy! Let us know your feedback if you have :)
